RecoHI/HiJetAlgos/python/HiRecoPFJets_cff.py

```diff
@@ -45,6 +45,17 @@
 akPu6PFJets = akPu5PFJets.clone(rParam       = cms.double(0.6), puPtMin = 30)
 akPu7PFJets = akPu5PFJets.clone(rParam       = cms.double(0.7), puPtMin = 35)
 
+ak4PFJetsForFlow = akPu5PFJets.clone(
+    Ghost_EtaMax = cms.double(5.0),
+    Rho_EtaMax = cms.double(4.4),
+    doRhoFastjet = cms.bool(False),
+    jetPtMin = cms.double(15.0),
+    nSigmaPU = cms.double(1.0),
+    rParam = cms.double(0.4),
+    radiusPU = cms.double(0.5),
+    src = cms.InputTag("pfcandCleaner", "particleFlowCleaned"),
+)
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.
```suggestion
ak4PFJetsForFlow = akPu5PFJets.clone(
    Ghost_EtaMax = 5.0,
    Rho_EtaMax = 4.4,
    doRhoFastjet = False,
    jetPtMin = 15.0,
    nSigmaPU = 1.0,
    rParam = 0.4,
    radiusPU = 0.5,
    src = ("pfcandCleaner", "particleFlowCleaned"),
)
```